File: chatwoot/csharp/src/OSEG.ChatwootExamples/UpdateAgentsInInboxExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class UpdateAgentsInInboxExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");

        var updateAgentsInInboxRequest = new UpdateAgentsInInboxRequest(
            inboxId: "inbox_id_string",
            userIds: [
            ]
        );

        try
        {
            var response = new InboxesApi(config).UpdateAgentsInInbox(
                accountId: 0,
                data: updateAgentsInInboxRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling InboxesApi#UpdateAgentsInInbox: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/UpdateAgentsInInboxExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
});

const addNewAgentToInboxRequest: api.AddNewAgentToInboxRequest = {
  inbox_id: "inbox_id_string",
  user_ids: [
  ],
};

new api.InboxesApi(configuration).updateAgentsInInbox(
  0, // accountId
  addNewAgentToInboxRequest, // data
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling InboxesApi#updateAgentsInInbox:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
